Non-canon Scarlet Crusade and Argent Dawn

Raged50 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- ### How to launch the event chain?
_AFTER_, Arthas goes down his canon storyline route and kills his own father, Uther the Lightbringer will get an event called "Kingslayer".
_If and only if_, Uther chooses to become rival with Arthas or Protecting Lordaeron in the "Kingslayer" event, only then the Scarlet Event chain will launch.

- ### Scarlet Event chains:
After the "Kingslayer" event, there will be meeting events for both Uther and those who come to meet Uther.
In this event chain, "Those that Remain", Human Rulers from across Lordaeron to Uther to attend his meeting where they can decide to convert to Scarletism or Argentism or Make an alliance with Uther Lightbringer.

- ### Scarlet Decisions:
After "Those that Remain", the player/AI can take the decisions to form the Scarlet Crusade and via decisions.
Scarlet Onslaught can be formed to journey to Northrend _after_ the Scarlet Crusade has been formed. (The Ruler of Scarlet Onslaught has the unique decision available to make a **New Heathglen** Special Building in Moaki)

- ### Argent Decisions:
After "Those that Remain", the player/AI can take the decisions to form the Argent Dawn and via decisions.

- ### Argent and Scarlet Common Decision:
Any **Human** of the **Holy Light**, **Scarletism** or **Argentism** Faiths can take the decision to "Liberate Lordaeron" and decide to whether to put themselves on the Throne of Lordaeron or choosing between a Living Menethil/Head of Faith. (NOTE: "Liberate Lordaeron" only becomes available _after_ Scarlet Crusade/Argent Dawn has been formed via the respective decisions)

**PLUS:** 
Ashbringer for 603 and 605 start dates!

Sally Whitemane/any other *female* Head of Faith for Scarletism should now be wearing the Whitemane Hat and Whitemane Armor

Scarlet and Argent Dawn helmets and armors for Scarletism and Argentism, respectively.

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Added this thing.
- Changed that thing.

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:

arithon committed:
Creation of the doomhammer (pics needed)

arithon committed:
removed frostmourne stuff

arithon committed:
values changed, rarity set to illustrous

arithon committed:
ashbringer made

arithon committed:
corrected typo in corrupted ashbringer's values

MemeWizard committed:
ashbringer icons

GDC

Chomski2 committed:
Merge branch 'master' into Ashbringers

Chomski2 committed:
Both ashbringers in game

Added Ashbringer
Added Corrupted ashbringer

skesus committed:
skesus fixed ashbringer icons

Fixed broken icons:
- Deleted "artifacts" folder in GFX and moved Ashbringer icons into the pre-existing "artifact" folder along with the other existing artifacts.
- Re-named Ashbringer Icon files to correct paths so they show up properly in-game

ElMariuso committed:
Merge branch 'dev' into Ashbringers

Raged50 committed:
global variable when terenas dies

Raged50 committed:
scarlet global pulse

Raged50 committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

Raged50 committed:
second limit not needed

Raged50 committed:
maybe this better yes

Raged50 committed:
scarlet setup and first event correctly fires

Raged50 committed:
scarlet event chain, causes big lag currently

Raged50 committed:
nor argument fix(?)

edmund-h commented:
converting this to draft as it's crashing due to a recursion issue. let's try to remember to add PRs as draft when they are up for public comment only : ]

Raged50 committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
Fixes

FelModder committed:
Moar fixes

FelModder committed:
Easier way to make scarlet events launch

FelModder committed:
ai_chances and LOC fixes

FelModder committed:
wrathful for scarletism and humble/calm for argentism

FelModder committed:
Error Log for the day

UDie2day committed:
Uther event fix

Allows the uther event and terenas funeral event to trigger when Arthas kills Terenas.

FelModder committed:
reverting some modifiers

FelModder committed:
ai_chances for Uther's events

FelModder committed:
changed what launches the scarlet event chains

FelModder committed:
Option C isn't useless and forms an alliance with Uther

FelModder committed:
event will trigger for geographical regions

FelModder committed:
Spawn troops for initial event

FelModder committed:
fixes

FelModder committed:
changing the description for alliance formation

UDie2day committed:
Alliance fix

Also pikemen -> pikemen_unit

FelModder committed:
More Alliance Fixes

FelModder committed:
last few description changes

FelModder committed:
Decision to Form the Scarlet Crusade and Head of Faith

FelModder committed:
Another event to notify all of Scarlet followers

FelModder committed:
LOC and event Fix for new Scarlet Leader

FelModder committed:
moar LOC fix

FelModder committed:
Sally will convert to Scarletism with Lordaeron's defeat

FelModder committed:
Female Head of Faith's get the sally whitemane appearance

FelModder committed:
Well, Sally herself get's her own look ofc

UDie2day committed:
Sally Whitemane HoF fix

FelModder committed:
Error Log fixes for the day

FelModder committed:
Fix needed for Sally Appearance

FelModder committed:
Basic Scarlet Onslaught decision and setup

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
small fix before continuing

FelModder committed:
Scarlet Onslaught Events (without desc) and Invasion Casus Belli

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
all these issues just because of brackets

FelModder committed:
more error log solving before continuing

FelModder committed:
That who formed the Scarlet Crusade, won't risk death in northrend

FelModder committed:
Scarlet Onslaught: Is the Lich King even alive rn?

FelModder committed:
New Hearthglen Decision, Building and Event

FelModder committed:
Update wc_modifiers_l_english.yml

FelModder committed:
Adding some modifiers for some more flavour

FelModder committed:
overrise_background for a few events

Lets use those background that mayonez_kun introduced while merging his mod

FelModder committed:
Argent Dawn Decision and Event

FelModder committed:
raise a new hearthglen fixes

FelModder committed:
Liberate Lordaeron basic set up

FelModder committed:
d_lordaeron will change faith and culture

Grekopithikos committed:
servent

FelModder committed:
Take Lordaeron for yourself, Give it to a Living Menethil or to your head of faith

FelModder committed:
Merge branch 'feat/code/scarlet-and-argent' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2 into feat/code/scarlet-and-argent

FelModder committed:
Fix LOC

FelModder committed:
ai_chances and options triggers

FelModder committed:
foxes fixes fuxes

FelModder committed:
maybe this will fix LOC

FelModder committed:
rename the LOC file

FelModder committed:
Last event's Description

FelModder committed:
Tweaking which Menethil is chosen

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
county level clash Casus Belli?

FelModder committed:
No the vassal only CB don't work

edmund-h commented:
i think the response to uther's call is bugged. i received some responses more than once

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,67 @@
+ SCARL.002_OPT_A:0 "Excellent."
+
+ SCARL.003_TITLE:1 "Those that Remain"
+ SCARL.003_DESC:0 "Lordaeron is lost. The dead rule the streets, and the Light's favour in us has faded.\nI will not march my people tom their deaths for a kingdom that no longer exists. We must look to other paths for our endurance.\nDo not seek us again."
```
```suggestion
 SCARL.003_DESC:0 "Lordaeron is lost. The dead rule the streets, and the Light's favour in us has faded.\nI will not march my people to their deaths for a kingdom that no longer exists. We must look to other paths for our endurance.\nDo not seek us again."
```

FelModder committed:
Tweaking costs and applying ai_goal

FelModder committed:
Only Independent and Uther's vassals are invited

FelModder committed:
fix the damn decision LOC

FelModder committed:
fix more decision LOC

FelModder committed:
Uther can now convert to Argentism as well

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
error log

FelModder committed:
not your own vassals

FelModder committed:
Reduce those requirements and fix LOC

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
Confederation Part 1

FelModder committed:
Any government can form cofederations

FelModder committed:
Custom tooltip, Notification event for Uther/Silver Hand Leader, Modifier for the Leader of Silver Hand and some reflavourization for the Call for Confederation event.

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
error log

FelModder committed:
change requirements for DH

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
In 603, Scarlet events will launch after 2/4 years

FelModder committed:
In 605, Scarlet events will launch after 1/2 years

Also changed the name of an event that was meant for uther but i changed it because it is no longer limited to just Uther

edmund-h committed:
Merge branch 'dev' into Ashbringers

edmund-h committed:
equip historical artifact to historical owner

edmund-h committed:
restrict usage and full benefit from certain chars

edmund-h committed:
add special effect and update loc

edmund-h committed:
fix artifact accessory genes

edmund-h committed:
fix special accessories file again???

edmund-h committed:
Merge branch 'feat/code/scarlet_extras' into feat/code/scarlet-and-argent

edmund-h committed:
error cleanup

edmund-h committed:
file consolidation

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
missing custom tooltip

FelModder committed:
error log

FelModder committed:
forgot to put in template?

FelModder committed:
these are not supposed to decay

FelModder committed:
change desc to fit 605 version

FelModder committed:
Adding Scarlet Armor

FelModder committed:
Adding Scarlet Helmet

FelModder committed:
MANY fixes...

FelModder committed:
No opinion worsening when you purge other cultures/religions

FelModder committed:
Make the Vassal directive with proper localizations (#1764)

FelModder committed:
error log

FelModder committed:
Argent Dawn Helmet

FelModder committed:
Argent Dawn Armor

FelModder committed:
helmet mesh fix

FelModder committed:
Merge branch 'dev' into feat/code/scarlet-and-argent

FelModder committed:
bookmark balnazzar fix

FelModder committed:
fixes part 1

FelModder committed:
fixes for liberate lordaeron

FelModder committed:
hopefullly whitemane look fix

FelModder committed:
what is this, the billionth fix?

Grekopithikos commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.007_OPT_A:0 "Send the word out for volunteers..."
+ SCARL.007_OPT_A_tt:0 "You send a message of your expedition to fellow Scarlets"
+
+ SCARL.008_TITLE:1 "I want YOU for Scarlet Army"
```
Should be "for the Scarlet Army"

FelModder replied:
i meant that as a "I want you for the US army meme" but i will change it

Grekopithikos replied:
Yeah I know I meant you missed a "the"

FelModder replied:
ahh okay, done

Grekopithikos commented on localization/english/wc_decisions_l_english.yml:
```diff
@@ -264,3 +261,34 @@
+ raise_a_new_hearthglen_decision_tooltip:0 "Hear me Roar!"
+
+ form_the_argent_dawn_decision:0 "Form the Argent Dawn"
+ form_the_argent_dawn_decision_desc:0 "The Paladins of the [GetTitleByKey('k_order_of_the_silver_hand').GetName] lie on the brink of destruction and the Scarlet wreak havoc in the name of righteousness... they are extremists who even punish their fellow Humans instead of helping them in a time of crises of faith and crises of existence itself. We will help the poor, we will help the needy, we wil ...
```
"crises of faith and crises of existence itself"
Should be
"crisis of faith and of existence itself"

FelModder replied:
done

FelModder committed:
Supposed to be outside hidden_effect

Grekopithikos commented on localization/english/wc_decisions_l_english.yml:
```diff
@@ -264,3 +261,34 @@
+ raise_a_new_hearthglen_decision_tooltip:0 "Hear me Roar!"
+
+ form_the_argent_dawn_decision:0 "Form the Argent Dawn"
+ form_the_argent_dawn_decision_desc:0 "The Paladins of the [GetTitleByKey('k_order_of_the_silver_hand').GetName] lie on the brink of destruction and the Scarlet wreak havoc in the name of righteousness... they are extremists who even punish their fellow Humans instead of helping them in a time of crises of faith and of existence itself. We will help the poor, we will help the needy, we will stand ...
```
"crisis" not "crises"

FelModder committed:
ai_check_interval is better

FelModder committed:
Liberate Lordaeron decision fixes

FelModder committed:
final few fixes for sally look

FelModder committed:
change game version with this

Grekopithikos committed:
maybe if it's more like pope hat?

FelModder committed:
error log

Grekopithikos committed:
Update 01_headgear_base.txt

FelModder committed:
Revert "Update 01_headgear_base.txt"

This reverts commit 497196af709e20fe684561dd3df11d77e9e64234.

FelModder committed:
Revert "maybe if it's more like pope hat?"

This reverts commit 0a3eb74bbf0eac75ac7b32f663329b4836d62b74.

FelModder committed:
rename this first

FelModder committed:
rename this as well

FelModder committed:
more error log

FelModder committed:
i hate if/else...

FelModder committed:
remove scarlet_mark if you form the scarlet crusade

edmund-h commented on localization/english/event_localization/wc_argent_events_l_english.yml:
```diff
@@ -0,0 +1,5 @@
+﻿l_english:
+
+ ARG.001_TITLE:1 "Order of the Argent Dawn"
+ ARG.001_DESC:0 "All the loyalist towards the Argent cause have been gathered! The forces of the Argent are no longer scattered.\n\nUnited by the will to save Lordaeron from collapse, the council and leaders of the [GetTitleByKey('e_argent_dawn').GetName] swore an oath to wipe any presence of evil from Lordaeron and help those in need."
```
```suggestion
 ARG.001_DESC:0 "All those loyal towards the Argent cause have been gathered! The forces of the merciful [GetFaithByKey('argent').HighGodName] are no longer scattered.\n\nUnited by the will to save Lordaeron from collapse, the council and leaders of the [GetTitleByKey('e_argent_dawn').GetName] swore an oath to wipe away the presence of evil from Lordaeron and help those in need."
```

edmund-h commented on localization/english/event_localization/wc_liberate_lordaeron_events_l_english.yml:
```diff
@@ -0,0 +1,11 @@
+﻿l_english:
+ 
+ liberate_lordaeron.1000.title:1 "Lordaeron Reborn"
+ liberate_lordaeron.1000.desc:0 "After countless deaths and conflicts, [GetTitleByKey('b_lordaeron').GetName] has been wrestled away from the Scourge's grasp.\n\nThe people of Lordaeron will settle back into their homelands and the plague shall be cleansed off... but now the choices are laid bare before us...\n\nWho shall inherit the new [GetTitleByKey('k_lordaeron').GetName]? I have taken back t ...
```
```suggestion
 liberate_lordaeron.1000.desc:0 "After countless deaths and conflicts, the [GetTitleByKey('b_lordaeron').GetName] has been wrested from the Scourge's grasp. The people of these ravaged lands will settle into their homelands once again and the plague shall be cleansed from the air and soil, but now the problem of rulership is laid bare before us... Who shall inherit the new [GetTitleByKey('b_lordaeron').GetBaseNameNoTierNoTooltip]?\n\nPerhaps the right thing to do would be to hand it back to the Menethils, to keep it stable? Or perhaps it would be most pious of me to hand it to the highest leadership of our Holy Faith?\n\nThen again, it was I alone who took back the [GetTitleByKey('b_lordaeron').GetTierAsNameNoTooltip] from the Scourge. After all, why shouldn't I keep it for myself?"
```
did a little gymnastics to preserve the LOTR reference

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.000_OPT_B:0 "I cannot afford to chase ghosts."
+ SCARL.000_OPT_tt:0 "You choose to stand alone."
+
+ SCARL.001_TITLE:1 "Those that Remain"
```
```suggestion
 SCARL.001_TITLE:1 "Those Who Remain"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.001_OPT_B:0 "This war is already lost."
+ SCARL.001_OPT_B_TT:0 "You ignore the call."
+
+ SCARL.002_TITLE:1 "Those that Remain"
```
```suggestion
 SCARL.002_TITLE:1 "$SCARL.001_TITLE$"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.002_DESC:0 "Your call has been heard. The Light has not abandoned us, nor will we abandon you. Lordaeron may be in ruins, but our duty still remains until our death. We will gather our forces and attend your meeting.\nFor Lordaeron, and for the Light."
+ SCARL.002_OPT_A:0 "Excellent."
+
+ SCARL.003_TITLE:1 "Those that Remain"
```
```suggestion
 SCARL.002_TITLE:1 "$SCARL.001_TITLE$"
```
or you could just use the same loc key for all three titles (might help prevent hash collisions)

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.003_OPT_A:0 "Cowards."
+
+ SCARL.004_TITLE:1 "Those that Remain"
+ SCARL.004_DESC:0 "You walk into the meeting room and quickly realise that the dimly lit hall is thick with tension. The remnants of the [GetTitleByKey('k_order_of_the_silver_hand').GetName] have gathered and what remains of their order has begun to splinter before your very eyes.\n\nOn one side of the room, knights in bloodstained armour speak with righteous fury - their voices sharp with anger. ...
```
```suggestion
 SCARL.004_DESC:0 "You walk into the meeting room and quickly realise that the dimly lit hall is thick with tension. The remnants of [GetTitleByKey('k_order_of_the_silver_hand').GetName] gathered here have begun to splinter before your very eyes.\n\nOn one side of the room, knights in bloodstained armour speak with righteous fury - their voices sharp with anger. #italic "We must cleanse [GetTitleByKey('k_lordaeron').GetBaseNameNoTier]! Purge the infected, root out the traitors— No mercy! No hesitation! [ROOT.Char.GetFaith.HighGodName] demands it! #!"\n\nOpposite to them stands a weary but resolute group, their conviction and faith in [ROOT.Char.GetFaith.HighGodName] being tempered by their compassion. #italic "The innocent still suffer," #! They argue in response. #italic "[ROOT.Char.GetFaith.HighGodName] does not demand cruelty! If we abandon them, we are no better than the monsters we fight." #!\n\nThe divide is clear:\nAll who have gathered here must make their choice.""
 SCARL.004_OPT_A:0 "The Light must be defended. No matter the cost."
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+
+ SCARL.0045_TITLE:1 "Last stand of the [GetTitleByKey('k_order_of_the_silver_hand').GetName]"
+ SCARL.0045_DESC:0 "By allying ourselves with what remains of the once powerful [GetTitleByKey('k_lordaeron').GetName], we have taken upon the task to protect the last vestiges of Lordaeron.\n\nIt seems that some didn't trust our call for a United front and have formed their own factions to stand up against the Lich King.\n\nWe shall stand tall against the Scourge... or maybe even become a powerh ...
+ SCARL.0045_OPT_A:0 "We shall Rise once again or Fall in vain"
```
```suggestion
 SCARL.0045_TITLE:1 "Last Stand of the [GetTitleByKey('k_order_of_the_silver_hand').GetName]"
 SCARL.0045_DESC:0 "By allying ourselves with what remains of the once powerful [GetTitleByKey('k_lordaeron').GetName], we have taken upon ourselves the protection of the last vestiges of [GetTitleByKey('k_lordaeron').GetBaseNameNoTierNoTooltip].\n\nAlthough it seems that some did not trust our call for a United front and have formed their own factions to stand up against the Lich King, 3e shall stand tall against the Scourge... Perhaps we may yet gather enough strength to take back our beloved home!"
 SCARL.0045_OPT_A:0 "We shall rise once again or fall in vain!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+
+ SCARL.006_TITLE:1 "Flames of the Remaining"
+ SCARL.006_DESC:0 "No longer are we standing alone against the Scourge. The neighbouring #italic Kings #!hide behind their walls instead of standing with the fellows of their own kind.\n\nOur leaders with the fires of zeal and vengeance have risen up! They have gathered the righteous and have formed a crusade against the scourge and those who would oppose us.\n\n[scarlet_leader.GetName] has been  ...
+ SCARL.006_OPT_A:0 "The Saviours are Here!"
```
```suggestion
 SCARL.006_TITLE:1 "Flames of the Remainders"
 SCARL.006_DESC:0 "No longer are we standing alone against the Scourge. The neighbouring #italic Lords #!hide behind their walls instead of standing with the fellowship of their own kind.\n\nOur leaders, alight with the fires of zeal and vengeance, have risen up! We have gathered the righteous and formed a permanent #bold Crusade #! against the Scourge and those who would oppose us.\n\n[scarlet_leader.GetName] has been chosen to lead us to victory! The [GetTitleByKey('k_lordaeron').GetName] will be born anew, when the Scourge and the Damned are burnt to ashes!"
 SCARL.006_OPT_A:0 "The Saviours are here!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.006_OPT_A:0 "The Saviours are Here!"
+
+ SCARL.007_TITLE:1 "Wrath of the Scarlets"
+ SCARL.007_DESC:0 "How many more of our brethren and our kind die or sacrificed before the Scourge is gone for good?\n\nHow long are we supposed to only dream about those peaceful times gone by?\n\nWe must take our fate in our own hands. We must cut the head of the snake. We must go to the darkest and icy depths of Northrend to slay the [GetTrait('lich_king').GetName(GetNullCharacter)] itself."
```
```suggestion
 SCARL.007_DESC:0 "How many more of our brethren and our kind shall die or be sacrificed before the Scourge is gone for good?\n\nHow long are we reduced to dreams of those peaceful times gone by?\n\nWe must take our fate in our own hands. We must cut off the head of the snake. We must go to the dark and icy depths of Northrend to slay the [GetTrait('lich_king').GetName(GetNullCharacter)] itself!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.007_OPT_A_tt:0 "You send a message of your expedition to fellow Scarlets"
+
+ SCARL.008_TITLE:1 "I want YOU for the Scarlet Army"
+ SCARL.008_DESC:0 "To the brave warriors loyal to the scarlet cause,\nHow has the progress against removing the scourge from [GetTitleByKey('k_lordaeron').GetName] going? Not very well I presume?\nThey are numerous and unending in all the senses... aren't they?\nJoin me. Join my expedition to slay the [GetTrait('lich_king').GetName(GetNullCharacter)]! Our only hope in winning for Humanity lies in ...
```
```suggestion
 SCARL.008_DESC:0 "To the brave warriors loyal to the Scarlet cause,\nHow has the progress against removing the Scourge from [GetTitleByKey('k_lordaeron').GetName] been going? Not very well I presume?\nThey are numerous and unending in every sense... aren't they? Our only hope in winning for Humanity lies in slaying the [GetTrait('lich_king').GetName(GetNullCharacter)]!\nJoin me. Join my expedition to $world_northrend$!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.008_TITLE:1 "I want YOU for the Scarlet Army"
+ SCARL.008_DESC:0 "To the brave warriors loyal to the scarlet cause,\nHow has the progress against removing the scourge from [GetTitleByKey('k_lordaeron').GetName] going? Not very well I presume?\nThey are numerous and unending in all the senses... aren't they?\nJoin me. Join my expedition to slay the [GetTrait('lich_king').GetName(GetNullCharacter)]! Our only hope in winning for Humanity lies in ...
+ SCARL.008_OPT_A:0 "This is madness!"
+ SCARL.008_OPT_B:0 "You are right... this is the only solution!"
```
```suggestion
 SCARL.008_OPT_B:0 "You are right... This is the only solution!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.008_OPT_B:0 "You are right... this is the only solution!"
+
+ SCARL.009_TITLE:1 "Foolish Endeavour"
+ SCARL.009_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou are either a fool or a bumbling idiot.\nGoing to the most icy hells where only wild creatures and the Scourge thrive? Where even the Light cannot give you enough warmth and hope?\nI wish you the best on your #italic "brave expedition" #!. I hope the Light blesses you nonetheless for your bravery.\nDo not concern us again."
```
```suggestion
 SCARL.009_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou are either a fool or a bumbling idiot.\nGoing to the most icy hells where only wild creatures and the Scourge thrive? Where even the Light can give you neither warmth nor hope?\nI wish you the best on your #italic "brave expedition" #! and hope the Light blesses you nonetheless for your bravery.\nDo not bother us again."
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.009_OPT_A:0 "Coward!"
+
+ SCARL.010_TITLE:1 "Brave Warrior"
+ SCARL.010_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou spoke my mind! The skirmishes across [GetTitleByKey('k_lordaeron').GetName] have, by far, done little to no progress...\nWe saw no other choice but to flee the kingdom as well but we are now ashamed of our behaviour!\nWe will atone for even letting such sinful thoughts flow inside us. We will join your expedition and crush the Scourge ...
```
```suggestion
 SCARL.010_TITLE:1 "Brave Warrior"
 SCARL.010_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou spoke my mind! The skirmishes across [GetTitleByKey('k_lordaeron').GetName] have thus far made little to no progress...\nWe saw no other choice but to flee the kingdom as well, but we are now ashamed of our decision!\nWe will atone for even letting such sinful thoughts flow inside us. We will join your expedition and crush the Scourge once and for all with our combined strength!"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.010_OPT_A:0 "Excellent..."
+
+ SCARL.011_TITLE:1 "Beginning of the End"
+ SCARL.011_DESC:0 "We have finally landed in [GetTitleByKey('c_moaki').GetName]. Very close to the [GetTrait('lich_king').GetName(GetNullCharacter)]'s lair but not within their grasp.\n\nFrom here we can launch our attacks on the Scourge and maybe even catch them by surprise?\n\nBut first, we will need to rid our new settlement from these... #italic animals #!that surround our new colony."
```
```suggestion
 SCARL.011_DESC:0 "We have finally landed in the [GetTitleByKey('c_moaki').GetName], within striking distance of the [GetTrait('lich_king').GetName(GetNullCharacter)]'s lair but not within its grasp.\n\nFrom here we can launch our attacks on the Scourge and, with [ROOT.Char.GetFaith.HighGodName]'s blessing, even catch them by surprise!\n\nBut first, we will need to rid our new colony of these... #italic animals#!... that surround our settlement."
```
keeping the pronoun consistent with "its"

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ scarlet_onslaught_troops:0 "Scarlet Armada"
+
+ SCARL.012_TITLE:1 "Beginning of the End"
+ SCARL.012_DESC:0 "[GetTitleByKey('c_moaki').GetName] is finally liberated from the filth that infected this place. We shall wipe the Scourge out from the face of Azeroth the same way!\n\nShall we keep expanding our base and secure more land and grow stronger? Or do we have enough to risk it against the [GetTrait('lich_king').GetName(GetNullCharacter)]?"
```
```suggestion
 SCARL.012_DESC:0 "With  [ROOT.Char.GetFaith.HighGodName]'s blessing, [GetTitleByKey('c_moaki').GetName] is finally liberated from the filth that infected this place. We shall wipe the Scourge clean from the face of Azeroth the same way!\n\nShall we keep expanding our base and secure more land and grow stronger? Or do we have enough forces to risk it against the [GetTrait('lich_king').GetName(GetNullCharacter)]?"
```

edmund-h commented on localization/english/event_localization/wc_scarlet_events_l_english.yml:
```diff
@@ -0,0 +1,72 @@
+ SCARL.012_OPT_A:0 "The end of the [GetTrait('lich_king').GetName(GetNullCharacter)] is nigh!"
+
+ SCARL.013_TITLE:1 "Our Pride, Our Hearthglen"
+ SCARL.013_DESC:0 "Those with the burning Light inside their heart, will no longer feel the cold and darkness of Northrend. We have raised our own Hearthglen, far away from our old homes. It gives us pride, it gives us joy, it gives us warmth.\n\nWith our Beacon of Light right behind us, we shall be guided against the Scourge and the heathens! Fellow [GetTrait('creature_human').GetName(GetNullCha ...
```
```suggestion
 SCARL.013_DESC:0 "Those with [ROOT.Char.GetFaith.HighGodName] burning inside their heart, will no longer feel the cold and darkness of $world_northrend$. We have raised our own New $c_hearthglen$, far away from our old home. It gives us pride, it gives us joy, it gives us warmth.\n\nWith our Beacon of Light shining behind us, we shall be guided forth against the Scourge and the heathens! Fellow [GetTrait('creature_human').GetName(GetNullCharacter)]s lost in the cold shall return to us by the guidance of the [ROOT.Char.GetFaith.HighGodName]!"
```

edmund-h commented on localization/english/modifiers/wc_modifiers_l_english.yml:
```diff
@@ -340,5 +340,35 @@
+ servant_of_lordaeron_desc:0 "By handing over the newly formed Kingdom of Lordaeron back to the Menethil bloodline, they have proven themselves to be a loyal servent of the Menethil."
+
+ warrior_of_lordaeron:0 "Warrior of Lordaeron"
+ warrior_of_lordaeron_desc:0 "As a pious servant of the Light, this character has proved themselves to be the most zealous of their faith by handing over the rulership of the new kingdom of Lordaeron to their Head of Faith."
```
```suggestion
 last_defender_of_lordaeron:0 "Last Leader of the Silver Hand"
 last_defender_of_lordaeron_desc:0 "By gathering allies and starting a flame within the citizens of Lordaeron, this character has taken the oath to defend the Remnants of the Kingdom to the last man."

 # Scarlet Story Modifiers
 wc_scarlet_crusader:0 "Scarlet Crusader"
 wc_scarlet_crusader_desc:0 "This character has taken an oath to wipe out the presence of the Scourge in Lordaeron and any others who stand in their way."

 wc_scarlet_zealot:0 "Scarlet Extremist"
 wc_scarlet_zealot_desc:0 "Driven either driven by pure zeal and dedication to the worship of the Light or by mere madness, this character has taken it upon themselves to end the Scourge by going against the Lich King himself."

 wc_building_new_hearthglen:0 "Building the New Capital"
 wc_building_new_hearthglen_desc:0 "Our people work hard and fast to fulfill their task of building our new Base. The New Hearthglen will be the evidence of our glory and our resolve."

 # Argent Story Modifiers
 wc_argent_leader:0 "Argent Crusader"
 wc_argent_leader_desc:0 "This character has taken the oath to protect the needy, wage war against the forces of evil and be compassionate to all sentient beings of Azeroth."

 # Liberate Lordaeron Modifiers

 saviour_of_lordaeron:0 "Saviour of Lordaeron"
 saviour_of_lordaeron_desc:0 "This character has wiped the presence of the Scourge from Lordaeron and has now declared themselves to be the ruler of the new kingdom that has risen from the ashes."

 servant_of_lordaeron:0 "Servant of Lordaeron"
 servant_of_lordaeron_desc:0 "By handing over the newly formed Kingdom of Lordaeron back to the Menethil bloodline, they have proven themselves to be a loyal servent of the House of Menethil."

 warrior_of_lordaeron:0 "Warrior of Lordaeron"
 warrior_of_lordaeron_desc:0 "As a pious servant of the Light, this character has proved themselves to be the most zealous of their faith by handing over the rulership of the new kingdom of Lordaeron to their Head of Faith."
```

FelModder committed:
changes as requested

Grekopithikos commented on common/on_action/game_start.txt:
```diff
@@ -579,6 +579,21 @@ on_game_start = {
+			}
+			# Sally Whitemane Conversion
+			character:60572 = {
+				set_character_faith_with_conversion = faith:scarletism
```
I don't think it makes sense for her to convert before the scarlet crusade becomes a thing

FelModder replied:
if i don't, then it fucks up the Scarlet Crusade formation decision because -> the effect finds Sally, but if sally isn't a scarlet already, then the event gets bugged out and no HOF title is made

FelModder replied:
<img width="578" height="273" alt="proog" src="https://github.com/user-attachments/assets/61f064b6-e2ae-42a5-a3ff-eb5f047f1f3a" />

And from looking it up, it does seem like Sally aligns with extremist ideology ever since tragedy struck her

FelModder replied:
Again, can't change this otherwise it messes up HOF title creation

edmund-h replied:
i think u made a good call here. sadly we have to fudge the lore sometimes in order to make it fit ck3 mechanics

FelModder committed:
more LOC fixes

FelModder committed:
increase costs of some decisions

Grekopithikos reviewed (approved):
ok
